### PR TITLE
a redirection for / route

### PR DESCRIPTION
**Pull request by peterjah**

### CODE REVIEW

**Review by B-Naoufal — APPROVED** [review]
A nice initiative to have an index in each plugin, it will be needed.

File: api/wallet_api-v0.yml [hunk 1]
```diff
@@ -10,6 +10,23 @@ schemes:
   - http
 swagger: "2.0"
 paths:
+  /:
+    get:
+      operationId: index
```
Comment (gregLibert) [inline api/wallet_api-v0.yml:15]:
```suggestion
      operationId: defaultHTMLPage
```

File: api/wallet_api-v0.yml [hunk 2]
```diff
@@ -10,6 +10,23 @@ schemes:
...
+      produces:
+        - application/json
+        - text/javascript
+        - text/html
+        - text/css
+        - text/webp
+        - image/png
```
Comment (gregLibert) [inline api/wallet_api-v0.yml:22]:
```suggestion
```

File: api/wallet_api-v0.yml [hunk 3]
```diff
@@ -10,6 +10,23 @@ schemes:
...
+      responses:
+        "200":
+          description: Page found
+        "404":
+          description: Resource not found.
+          schema:
+            $ref: "#/definitions/Error"
```
Comment (gregLibert) [inline api/wallet_api-v0.yml:29]:
```suggestion
        "301":
          description: Move to the default endpoint
```

File: internal/handler/web.go [hunk 4]
```diff
@@ -33,3 +33,14 @@ func WebWalletHandler(params operations.WebParams) middleware.Responder {
...
+//nolint:nolintlint,ireturn
+func IndexHandler(params operations.IndexParams) middleware.Responder {
+
+	resourceContent, err := web.Content(indexHTML)
+	if err != nil {
+		return operations.NewIndexNotFound()
+	}
+
+	return helper.NewCustomResponder(resourceContent, map[string]string{}, http.StatusOK)
+}
```
Comment (gregLibert) [inline internal/handler/web.go:46]:
```suggestion
func IndexHandler(_ operations.DefaultPageParams) middleware.Responder {
	return helper.NewCustomResponder(nil, map[string]string{"Location": "web/index.html"}, http.StatusPermanentRedirect)
}
```

File: web/html/index.html [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<html>
```
Comment (gregLibert) [inline web/html/index.html:1]: Not needed anymore

**Review by gregLibert — CHANGES_REQUESTED** [review]
I think it's easier and cleaner to use a permanent redirect

**Review by gregLibert — APPROVED** [review]

**Pull request merged**